web/web_test.go

```diff
@@ -99,55 +99,37 @@ func TestReadyAndHealthy(t *testing.T) {
 	time.Sleep(5 * time.Second)
 
 	resp, err := http.Get("http://localhost:9090/-/healthy")
-	if err != nil {
-		t.Fatalf("Unexpected HTTP error %s", err)
-	}
-	if resp.StatusCode != http.StatusOK {
-		t.Fatalf("Path /-/healthy with server unready test, Expected status 200 got: %s", resp.Status)
-	}
+
+	testutil.Ok(t, err)
+	testutil.Equals(t, resp.StatusCode, http.StatusOK)
 
 	resp, err = http.Get("http://localhost:9090/-/ready")
-	if err != nil {
-		t.Fatalf("Unexpected HTTP error %s", err)
-	}
-	if resp.StatusCode != http.StatusServiceUnavailable {
-		t.Fatalf("Path /-/ready with server unready test, Expected status 503 got: %s", resp.Status)
-	}
+
+	testutil.Ok(t, err)
+	testutil.Equals(t, resp.StatusCode, http.StatusServiceUnavailable)
 
 	resp, err = http.Get("http://localhost:9090/version")
-	if err != nil {
-		t.Fatalf("Unexpected HTTP error %s", err)
-	}
-	if resp.StatusCode != http.StatusServiceUnavailable {
-		t.Fatalf("Path /version with server unready test, Expected status 503 got: %s", resp.Status)
-	}
+
+	testutil.Ok(t, err)
+	testutil.Equals(t, resp.StatusCode, http.StatusServiceUnavailable)
```

Comment:
I think Equals is `(t, expected, actual)`

Reply:
D'uh, went too fast, sorry about that.